.github/workflows/test.yml

```diff
@@ -0,0 +1,55 @@
+name: Test
+
+on:
+  push:
+    branches: [ core ]
+  pull_request:
+    branches: [ core ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
20.04 sounds a better bet for something aimed at servers

Reply:
good point! 
changed at https://github.com/rubysherpas/paranoia/pull/515/commits/11908cdc36fbe35a0ce21888e20f4fab21e3dd57